ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Table/ScanConfigsTablePage.tsx

```diff
@@ -190,8 +195,7 @@ function ScanConfigsTablePage({
                         <ToolbarContent>
                             <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
                                 <Pagination
-                                    isCompact
-                                    itemCount={scanSchedules ? scanSchedules.length : 0}
+                                    itemCount={scanSchedulesCount ?? 0}
```

Comment:
@dvail Remind me, does `useRestQuery` hook always need optional chaining for `data` property?

It looks like yes:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useRestQuery.ts#L15

Asking, not necessarily suggesting, what are pro and con of optional argument for initial value, like `0` in this case?

Reply:
I agree. An optional "default value" parameter seems like it would be worthwhile in a few instances. I don't really see any downsides as long as it's used carefully